tmt/steps/execute/__init__.py

```diff
@@ -77,10 +81,10 @@ class ScriptCreatingFile(Script):
 
 # Script handling reboots, in restraint compatible fashion
 TMT_REBOOT_SCRIPT = ScriptCreatingFile(
-    path=Path("/usr/local/bin/tmt-reboot"),
+    path=Path(f"{SCRIPTS_DEST_DIR}/tmt-reboot"),
     aliases=[
-        Path("/usr/local/bin/rstrnt-reboot"),
-        Path("/usr/local/bin/rhts-reboot")],
+        Path(f"{SCRIPTS_DEST_DIR}/rstrnt-reboot"),
```

Comment:
Once `SCRIPTS_DEST_DIR` becomes `Path`, this would turn into `SCRIPTS_DEST_DIR / 'rstrnt-reboot'`

Reply:
moved all occurrences using `Path`